organizational/position_roles.rst

```diff
@@ -0,0 +1,40 @@
+Position Roles 
+========================
+
+Below are the positions that have allowed Hacker's Society function in the past. The
+listed positions are setup to run ACM and IEEE under the umbrella of Hacker's Society.
+The given descriptions are recommendations are able to be changed as the needs of the
+organization change. Along with that additional positions can be added when need.
+
+**President**
+
+- Runs general body meetings
+- Manages the organization ensuring others accomplish specified tasks
+
+**Vice President of ACM**
+
+- Maintain engagement with `ACM <http://www.acm.org/>`_
+- Keep involvment with CS/CE portion of the department
```

Comment:
We're forgetting the oft-forgotten fourth major in EECS. Systems and control. Not sure if they fall under ACM or IEEE